APKBUILD.5

```diff
@@ -0,0 +1,488 @@
+.Dd February 13, 2018
+.Dt APKBUILD 5 PRM
+.Os "Alpine Linux"
+
+
+.Sh NAME
+.Nm APKBUILD
+.Nd metadata and instructions to build a package
+
+
+.Sh SYNOPSIS
+.Nm /usr/src/packages/<repo>/<package>/APKBUILD
+
+
+.Sh DESCRIPTION
+An
+.Nm
+file is used by tools such as
+.Xr abuild 1
+to build a package for eventual installation by the
+.Xr apk 8
+package manager.  It defines metadata such as the name of the package, the
+version information, the source license, and contact information for the
+developer.  It additionally contains the commands needed to build, test, and
+install the package.
+.Pp
+The
+.Nm
+format is similar to a typical shell script; you set pre-defined variables and
+implement pre-defined functions, and the
+.Xr abuild 1
+(or similar) utility will use them to create the package.
+
+
+.Ss Required Variables
+The following variables must be set in all
+.Nm
+files:
+
+.Bl -tag -width Ds
+.It Cm pkgname
+Specifies name of the package.  This is typically the name of the package
+upstream; however, note that all letters must be lowercased.
+.Pp
+Libraries for scripting languages should have a prefix before the library name
+describing the language.  Such prefixes include
+.Em lua- ,
+.Em perl- ,
+.Em py- ,
+and
+.Em rb- .
+Not all languages use prefixes.  For a definitive list, consult the PREFIXES
+file in the root directory of the repository you are using for packaging.
+
+.It Cm pkgver
+Specifies the version of the software being packaged.  The version specified
+must match the pattern
+.Li digit[.digit]*[_suffix[digit]]*
```

Comment:
This is incomplete. We use exactly this https://devmanual.gentoo.org/ebuild-writing/file-format/index.html just with more suffixes.

Reply:
It might be worth noting, that some suffix values indicate a higher version than the one without the suffix, and some a lower version. Recently I've extended the [APKBUILD_Reference#pkgver](https://wiki.alpinelinux.org/wiki/APKBUILD_Reference#pkgver) documentation based on the `apk-tools` source to list that information.